salt/metalk8s/orchestrate/bootstrap/init.sls

```diff
@@ -170,3 +172,13 @@ Deploy MetalK8s UI:
     - saltenv: {{ saltenv }}
     - require:
       - salt: Precheck for MetalK8s UI
+
+Store MetalK8s version in annotations:
+  metalk8s_kubernetes.namespace_annotation_present:
+    - name: "kube-system"
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+    - annotation_key: "metalk8s.scality.com/version"
```

Comment:
Maybe this should be `metalk8s.scality.com/cluster-version` to be a bit more explicit. WDYT @scality/metalk8s

Reply:
fixed now